### PR TITLE
Remove scheduling.v1alpha1 and scheduling.v1alpha2 API

### PR DESCRIPTION
**Pull request by thandayuthapani**
Fixes https://github.com/volcano-sh/volcano/issues/692

`Remove scheduling.v1alpha1 and scheduling.v1alpha2 API`

### CODE REVIEW

**Comment by k82cn** [comment]
LGTM overall, just one tiny comment :)

**Comment by k82cn** [comment]
Thanks very much to you! We need to get it merged until next week to make sure everyone aware this API change.

File: pkg/apis/scheduling/v1beta1/types.go [hunk 1]
```diff
@@ -262,6 +262,9 @@ type QueueSpec struct {
 	Weight     int32           `json:"weight,omitempty" protobuf:"bytes,1,opt,name=weight"`
 	Capability v1.ResourceList `json:"capability,omitempty" protobuf:"bytes,2,opt,name=capability"`
 
+	// State controller the status of queue
+	State QueueState `json:"state,omitempty" protobuf:"bytes,3,opt,name=state"`
```
[the remark this answers is not in the document]
Comment (thandayuthapani) [reply pkg/apis/scheduling/v1beta1/types.go]: Will make those changes and raise new PR

**Pull request merged**